packages/core/src/utils/localizeEntity.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { describe, it, expect } from 'vitest';
+import { localizeEntity } from './localizeEntity';
+import { Entry } from 'contentful';
+
+// filepath: /Users/thomas.kellermeier/work/sparks/experience-builder/packages/core/src/utils/localizeEntity.test.ts
```

Comment:
Is this leftover comment or do we need it?

Reply:
Good catch! Don't know how that ended up in here. I'll delete it.